x-pack/plugins/cases/public/components/case_view/components/case_view_activity.tsx

```diff
@@ -179,15 +197,42 @@ export const CaseViewActivity = ({
       ? [caseUsers.reporter]
       : [convertToCaseUserWithProfileInfo(caseData.createdBy)];
 
+  const handleUserActionsActivityChanged = useCallback(
+    (params: UserActivityParams) => {
+      setUserActivityQueryParams({ type: params.type, sortOrder: params.sortOrder });
+    },
+    [setUserActivityQueryParams]
+  );
+
+  const isLoadingDescription = isLoading && loadingKey === 'description';
+
   return (
     <>
       <EuiFlexItem grow={6}>
+        <CaseViewTabs caseData={caseData} activeTab={CASE_VIEW_PAGE_TABS.ACTIVITY} />
+        <DescriptionWrapper
+          isLoadingDescription={isLoadingDescription}
+          data={caseData}
+          userProfiles={userProfiles}
+          onUpdateField={onUpdateField}
+        />
+        <EuiSpacer size="l" />
+        <EuiFlexItem grow={false}>
+          <UserActionsActivityBar
+            onUserActionsActivityChanged={handleUserActionsActivityChanged}
+            params={userActivityQueryParams}
+            userActionsStats={userActionsStats}
+            isLoading={isLoadingUserActionsStats}
+          />
+        </EuiFlexItem>
+        <EuiSpacer size="l" />
         {(isLoadingUserActions || isLoadingCaseConnectors) && (
-          <EuiLoadingContent lines={8} data-test-subj="case-view-loading-content" />
+          <EuiSkeletonText lines={8} data-test-subj="case-view-loading-content" />
```

Comment:
Thanks for doing this! The `EuiSkeletonText` accepts an `isLoading` prop. When is `false` it will render all of its children. We can do:

```
<EuiSkeletonText
          lines={8}
          data-test-subj="case-view-loading-content"
          isLoading={isLoadingUserActions || isLoadingCaseConnectors}
        >
          <EuiFlexGroup direction="column" responsive={false} data-test-subj="case-view-activity">
            <EuiFlexItem>
              {showUserActions && (
                <UserActions
                  userProfiles={userProfiles}
                  currentUserProfile={currentUserProfile}
                  getRuleDetailsHref={ruleDetailsNavigation?.href}
                  onRuleDetailsClick={ruleDetailsNavigation?.onClick}
                  caseConnectors={caseConnectors}
                  caseUserActions={userActionsData.userActions}
                  data={caseData}
                  actionsNavigation={actionsNavigation}
                  isLoadingUserActions={isLoadingUserActions}
                  onShowAlertDetails={onShowAlertDetails}
                  onUpdateField={onUpdateField}
                  statusActionButton={
                    permissions.update ? (
                      <StatusActionButton
                        status={caseData.status}
                        onStatusChanged={changeStatus}
                        isLoading={isLoading && loadingKey === 'status'}
                      />
                    ) : null
                  }
                  filterOptions={userActivityQueryParams.type}
                  useFetchAlertData={useFetchAlertData}
                />
              )}
            </EuiFlexItem>
          </EuiFlexGroup>
        </EuiSkeletonText>
```